package.json

```diff
@@ -112,7 +113,7 @@
         "build:prod": "BABEL_ENV=production NODE_ENV=production node --max_old_space_size=4096 ./node_modules/webpack/bin/webpack.js --progress --colors --config build/webpack.config.js",
         "clean": "rm -rf dist && rm -rf reports/coverage && rm -rf src/i18n/json",
         "cy:open": "yarn cy:wait; yarn cypress open",
-        "cy:run": "yarn cy:wait; yarn cypress run --spec test/integration/**/*.test.js",
+        "cy:run": "yarn cy:wait; yarn cypress run --spec test/integration/**/*.cy.js",
```

Comment:
Can we add the `specPattern` option in our Cypress config file rather than renaming all our files?